wordpress-mobile/WordPress-Android
# Issue/7545 chromeos back button

Fixes #7545 

To test:

In a Chromebook, press the back button when in the main activity and make sure it doesn't close the app. The back button should work like a normal back button in every other scenario.
